fetch.bs

```diff
@@ -2371,6 +2382,32 @@ is a <a>filtered response</a> whose
  <a attribute for=Response lt=ok><code>response.ok</code></a>, will return rather useless results.
 </div>
 
+<div class=example id=example-filtered-responses>
+ <p>The <a for=response>type</a> of a <a for=/>response</a> is exposed to script through the
+ {{Response/type}} getter:
+
+ <pre><code class=lang-javascript>
+(async function filteredResponseDemo() {
```

Comment:
You can assume top-level await now, IMO, so the wrapper is no longer necessary :).